htdocs/public/project/new.php

```diff
@@ -211,21 +211,21 @@ function llxFooterVierge()
 
 	if (!$error) {
 		// Search thirdparty and set it if found to the new created project
-		$result = $thirdparty->fetch(0, '', '', '', '', '', '', '', '', '', $object->email);
+		$result = $thirdparty->fetch(0, '', '', '', '', '', '', '', '', '', $GETPOST('email', 'alpha'));
```

Comment:
$GETPOST ? Are you sure ?
Also why adding 'alpha' as parameter of GETPOST instead of default value ?

Reply:
Oops!
Of course, only GETPOST() is sufficient.
I added 'alpha' out of habit, but in fact, the default parameter is sufficient too.